test/e2e/templates/basic-cluster.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   clusterName: cluster
   dnsPrefix: basic-cluster-dns
   imported: false
-  kubernetesVersion: 1.24.6
+  kubernetesVersion: 1.25.5
```

Comment:
I think we need to update the kind image version we use to set up the kind cluster to make it aligned since it is using the same versions: https://github.com/rancher/aks-operator/blob/ee6e4a03ebf58cb752ff2bcb81f5f65f82f601e3/Makefile#L10 and https://github.com/rancher/aks-operator/blob/9cef3583b32390099a7a7d3a1477cb05b445d59b/scripts/setup-kind-cluster.sh#L3

wdyt?

Reply:
These versions can be different but it's a good idea to bump kind too. Thanks for spotting it.

Reply:
done